contracts/feature-tests/contract-base-tester/mandos/adder.scen.json

```diff
@@ -0,0 +1,99 @@
+{
```

Comment:
You don't actually need additional tests for this task, if anything is wrong it will fail at compilation.

Reply:
You're right! Done